src/tests/backend/specs/api/pad.js

```diff
@@ -586,6 +588,31 @@ describe(__filename, function () {
     });
   });
 
+  describe('appendText', function () {
+    it('Append text to a pad Id', function (done) {
+      agent.get(`${endPoint('appendText', '1.2.13')}&padID=${testPadId}&text=hello`)
+          .expect((res) => {
+            if (res.body.code !== 0) throw new Error('Pad Append Text failed');
+          })
+          .expect('Content-Type', /json/)
+          .expect(200, done);
+    });
```

Comment:
Use Promises and `assert`:

```suggestion
    it('Append text to a pad Id', async function () {
      const res = await agent.get(`${endPoint('appendText', '1.2.13')}&padID=${testPadId}&text=hello`)
          .expect(200)
          .expect('Content-Type', /json/);
      assert.equal(res.body.code, 0);
    });
```